packages/seacas/libraries/ioss/src/text_mesh/Iotm_TextMeshUtils.h

```diff
@@ -14,593 +14,1754 @@
 #include <vector>                                    // for vector
 #include <unordered_map>
 #include <sstream>                       // for ostringstream
+#include <iostream>
 #include <functional>
 #include <stdexcept>
+#include <numeric>
+#include <tokenize.h>
+#include <fmt/ostream.h>
+#include <strings.h>
 
 // clang-format on
 // #######################   End Clang Header Tool Managed Headers  ########################
 namespace Iotm {
-  namespace text_mesh {
-    using ErrorHandler = std::function<void(const std::ostringstream &)>;
-
-    template <class EXCEPTION> void handle_error(const std::ostringstream &message)
-    {
-      throw EXCEPTION((message).str());
+namespace text_mesh
+{
+using ErrorHandler = std::function<void(const std::ostringstream &)>;
+
+template <class EXCEPTION>
+void handle_error(const std::ostringstream &message)
+{
+  throw EXCEPTION((message).str());
+}
+
+inline void default_error_handler(const std::ostringstream &message)
+{
+  handle_error<std::logic_error>(message);
+}
+
+inline std::string strip(const std::string &inpt)
+{
+  auto start_it = inpt.begin();
+  auto end_it = inpt.rbegin();
+  while (std::isspace(*start_it)) ++start_it;
+  while (std::isspace(*end_it)) ++end_it;
+  return std::string(start_it, end_it.base());
+}
+
+inline std::vector<std::string> tokenize(const std::string &str, const std::string &separators)
```

Comment:
Would be better to use the existing IOSS tokenize function in `tokenize.h` to avoid duplicate code.

Reply:
Please see the first comment on independent portability. The tokenize function was taken from tokenize.h and modified slightly for TextMesh use

Reply:
If this is truly needed, then we should remove the `#include <tokenize.h>` in Iotm_TextMeshUtils.h and Iotm_TextMesh.C and probably rename it to make it very clear which tokenize is being used.

Reply:
Good point